arm/ripper/utils.py

```diff
@@ -60,6 +62,18 @@ def scan_emby(job):
     else:
         logging.info("EMBY_REFRESH config parameter is false.  Skipping emby scan.")
 
+def SleepCheckProcess(ProcessStr, Proc_Count):
+
+    if Proc_Count != 0:
+        Loop_Count = Proc_Count + 1
+        logging.debug("Loop_Count " + str(Loop_Count))
+        logging.info("Starting A sleep check of " + str(ProcessStr))
+        while Loop_Count >= Proc_Count:
+            Loop_Count = sum(1 for proc in psutil.process_iter() if proc.name() == ProcessStr)
+            logging.debug("Number of Processes running is:" + str(Loop_Count) + " going to waiting 12 seconds.")
+            time.sleep(10)
```

Comment:
Not blocking, but should probably be configurable.

Reply:
Which part? The Duration of the sleep command? The reason I spaced it out to 10 ms was to reduce compute cycles.